tests/inference/test_query_single.py

```diff
@@ -0,0 +1,77 @@
+import subprocess
+import pytest
+
+# Config matrix
+# models_array = ["gpt2", "gpt2 gpt-j-6b", "gpt2 bloom-560m", "falcon-7b"]
+# model_endpoint_array = ["http://127.0.0.1:8000", None]
+# streaming_response_array = [True, False]
+# max_new_tokens_array = [10, None]
+# temperature_array = [0.7, None]
+# top_p_array = [0.6, None]
+# top_k_array = [5, None]
+
+# Temp Config matrix
+models_array = ["gpt2"]
+model_endpoint_array = ["http://127.0.0.1:8000"]
+streaming_response_array = [False]
+max_new_tokens_array = [10]
+temperature_array = [0.7]
+top_p_array = [0.6]
+top_k_array = [5]
+
+
+# Parametrize the test function with different combinations of parameters
+@pytest.mark.parametrize(
+    "models, model_endpoint, streaming_response, max_new_tokens, temperature, top_p, top_k",
+    [
+        (models, model_endpoint, streaming_response, max_new_tokens, temperature, top_p, top_k)
+        for models in models_array
```

Comment:
I think you can use list directly without defining another models_array above. it's the same and more clearer.

Reply:
done

Reply:
> done

just need to click Resolve conversation when done.